src/api/fungible_asset.ts

```diff
@@ -36,4 +46,48 @@ export class FungibleAsset {
   }): Promise<GetFungibleAssetMetadataResponse> {
     return getFungibleAssetMetadata({ aptosConfig: this.config, ...args });
   }
+
+  /**
+   * Queries the fungible asset activities
+   *
+   * This query returns the fungible asset activities.
+   * It can be filtered by owner address, asset type, and type.
+   *
+   * @param options.pagination Where offset and limit can be specified
+   * @param options.whereCondition Where condition to filter the results
+   * @returns getFungibleAssetMetadata A list of fungible asset metadata
+   */
+  async getFungibleAssetActivities(args: {
+    options?: {
+      pagination?: PaginationArgs;
+      whereCondition?: {
```

Comment:
same comment as in the other PR https://github.com/aptos-labs/aptos-ts-sdk/pull/78/files#r1361610560

Reply:
Yep removed and converted to using bool exp